### ISSUE TITLE
the questions about autoregressive models 关于自回归模型的问题

### ISSUE THREAD

**Issue by WGS-note** [yhw-yhw/TalkSHOW#25]
大家好，请教大家问题：

我在训练pixel自回归模型时，遇到以下两个问题：
When I was training a pixel autoregressive model, I encountered the following two problems:

1. gated pixelcnn 非常容易过拟合，在1、2个epoch之后，val loss就一直上升不降。
   Gated pixelcnn is very easy to overfit. After 1 or 2 epochs, the val loss keeps rising.
2.自回归模型会出现身份泄露的问题，比如speakerA生成时，会出现speakerB的动作和手势。
   The autoregressive model will have the problem of identity leakage. For example, when speakerA is generated, the actions and gestures of speakerB will appear.

请教大家有什么解决问题的策略？

@yhw-yhw @feifeifeiliu
